src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -8,6 +8,10 @@
 	</PropertyGroup>
 	<PropertyGroup>
     <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
You have introduced some breaking changes, please update the minor version here.

Reply:
In fact, 0.19.0-preview is the new version. It includes all the breaking changes, i.e. the changes are breaking with respect to our previous version 0.18.1-preview.

Reply:
This SDK generation corresponds to following changes in swagger spec:
https://github.com/Azure/azure-rest-api-specs/commit/d10a7a137eb2a2c13b63358091d6af90aa9f1674

https://github.com/Azure/azure-rest-api-specs/commit/06fac6ef2e98d9f321735138125aef7598b28ddb

https://github.com/Azure/azure-rest-api-specs/commit/b4fc5be0e8aded9ebd8b2099e416d55a3c432ea3